website/content/en/getting-started/installation/_index.md

```diff
@@ -0,0 +1,145 @@
+
+#### Using Deployment yamls
+
+1. **Install the Secrets Store CSI Driver**
+
+    💡 Follow the [Installation guide for the Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver#usage) to install the driver.
+
+    > **NOTE:** `0.0.9+` release of the Azure Key Vault provider is incompatible with the Secrets Store CSI Driver versions < `v0.0.14`. While installing the Secrets Store CSI Driver using yamls, add the following flag `--grpc-supported-providers=azure` to the [Linux](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/deploy/secrets-store-csi-driver.yaml) and [Windows](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/deploy/secrets-store-csi-driver-windows.yaml) *daemonset* manifests.
+    > - The flag needs to be added to the `secrets-store` container args
+
+    <details>
+    <summary>Result</summary>
+
+    ```
+    csidriver.storage.k8s.io/secrets-store.csi.k8s.io created
+    serviceaccount/secrets-store-csi-driver created
+    clusterrole.rbac.authorization.k8s.io/secretproviderclasses-role created
+    clusterrolebinding.rbac.authorization.k8s.io/secretproviderclasses-rolebinding created
+    clusterrole.rbac.authorization.k8s.io/secretprovidersyncing-role created
+    clusterrolebinding.rbac.authorization.k8s.io/secretprovidersyncing-rolebinding created
+    daemonset.apps/csi-secrets-store-windows created
+    daemonset.apps/csi-secrets-store created
+    customresourcedefinition.apiextensions.k8s.io/secretproviderclasses.secrets-store.csi.x-k8s.io created
+    customresourcedefinition.apiextensions.k8s.io/secretproviderclasspodstatuses.secrets-store.csi.x-k8s.io created
+    ```
+
+    </details><br/>
+
+    To validate the driver is running as expected, run the following command:
+
+    ```bash
+    kubectl get pods -l app=csi-secrets-store -n kube-system
+    ```
+
+    You should see the driver pods running on each agent node:
+
+    ```bash
+    NAME                      READY   STATUS    RESTARTS   AGE
+    csi-secrets-store-bp4f4   3/3     Running   0          24s
+    ```
+
+    To validate the `--grpc-supported-providers=azure` arg has been configured correctly, run the following command:
+
+    ```bash
+    kubectl get ds -l app=csi-secrets-store -o jsonpath='{range .items[*]}{.spec.template.spec.containers[1].args}{"\n"}'
+    ```
+
+    You should see the args for the `secrets-store` container in the driver pods for each node:
+    ```bash
+    ["--debug=true","--endpoint=$(CSI_ENDPOINT)","--nodeid=$(KUBE_NODE_NAME)","--provider-volume=/etc/kubernetes/secrets-store-csi-providers","--grpc-supported-providers=azure","--metrics-addr=:8080"]
+    ```
+
+2. **Install the Azure Key Vault provider**
+
+    For linux nodes
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/0.0.11/deployment/provider-azure-installer.yaml
```

Comment:
Do we want to hard code `0.0.11` here? does that mean we have to change these docs after every release?

Reply:
With versioned url the user knows which version they're installing/uninstalling with the deployment manifests. This would mandate an update during the release. When we update the manifests, we'll also need to update the docs prior to release. WDYT?

Reply:
I was thinking we can avoid having to update the docs after every new release by pointing the yamls to the latest chart and deployment folder and make a note that this is going to deploy the latest release version. wdyt?

Reply:
sounds good! I'll update it.

Reply:
@ritazh Updated. PTAL!